cpp/include/cugraph/algorithms.hpp

```diff
@@ -1822,6 +1825,66 @@ uniform_nbr_sample(raft::handle_t const& handle,
                    bool with_replacement = true,
                    uint64_t seed         = 0);
 
+/**
+ * @brief Uniform Neighborhood Sampling.
+ *
+ * This function traverses from a set of starting vertices, traversing outgoing edges and
+ * randomly selects from these outgoing neighbors to extract a subgraph.
+ *
+ * Output from this function a set of tuples (src, dst, edge_id, edge_type, weight, hop, label),
```

Comment:
`Output from this function a set of tuples`=>`Output from this function is a tuple of vectors`?